packages/react-scripts/scripts/init.js

```diff
@@ -53,7 +53,7 @@ function tryGitInit(appPath) {
     didInit = true;
 
     execSync('git add -A', { stdio: 'ignore' });
-    execSync('git commit -m "Initial commit from Create React App"', {
+    execSync('git commit -m ":tada: Initial commit from Create React App"', {
```

Comment:
Shouldn't this be a 🎉 symbol? `:tada:` relies on the GitHub parser to get converted to an emoji.

Reply:
There’s a discussion about this in the gitmoji project. https://github.com/carloscuesta/gitmoji/issues/60#issuecomment-276158959. @carloscuesta prefers using the short codes so I’ve used that system. I’m a big fan of his project so would quite like to follow its, very loose, guidelines.

Reply:
I understand, but doing so, systems different from GitHub will just display the plain text if I'm not wrong. This project is aimed to a wider audience.

Reply:
Yeah, that’s fair enough. Happy to change if the maintainers prefer Unicode.